libcloud/dns/drivers/powerdns.py

```diff
@@ -182,31 +189,47 @@ def create_record(self, name, zone, type, data, extra=None):
                                              zone.id)
         if extra is None or extra.get('ttl', None) is None:
             raise ValueError('PowerDNS requires a ttl value for every record')
-        record = {
+
+        record_name = self._fmt_name(name)
+        id = ':'.join((self.RECORD_TYPE_MAP[type], record_name))
```

Comment:
Would you please add some explanation in the documentation about this id change? I used "None" for a reason - PowerDNS itself does not expose IDs for records (via the HTTP API, anyway).